pkg/api/api_test.go

```diff
@@ -0,0 +1,20 @@
+package api_test
+
+import (
+	"testing"
+
+	"github.com/hellofresh/janus/pkg/api"
+	"github.com/stretchr/testify/assert"
+)
+
+const (
+	testKey   = "key"
+	testValue = "value"
+)
+
+func TestNewInstanceOfDefinition(t *testing.T) {
```

Comment:
IntelliJ Gogland suggest the following names for tests (and I like it, although it is not necessary):
* `Test<FunctionName>()` for functions
* `Test<StructName>_<MethodName>()` for struct methods

With the naming like this it is easier to track what it tests.